src/Germany/README.md

```diff
@@ -90,9 +92,11 @@ Example:
 ```csharp
 var transactionId = Guid.NewGuid();
 var startedTransaction = await client.StartTransactionAsync(accessToken, clientId, tssId, transactionId);
-var endedTransaction = await client.FinishTransactionAsync(accessToken, clientId, tssId, InvoiceToReport, transactionId, lastRevision: "1");
+var endedTransaction = await client.FinishTransactionAsync(accessToken, clientId, tssId, InvoiceToReport, transactionId, revision: 2);
```

Comment:
Don't forget to remove the revision from readme in case we decide to actually do that. I commented it in the PR, but we didn't resolve it yet.